.env.example

```
@@ -0,0 +1,26 @@
+# Copy this file to .env at the repo root and adjust as needed for local development.
+# `make up` / `docker compose` picks this up automatically via --env-file .env.
+
+# --- MariaDB ---
+DB_ROOT_PASSWORD=root
+DB_NAME=app_development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_ROOT_PASSWORD key

(UnorderedKey)